bin/live/pycbc_live_single_trigger_fits

```diff
@@ -273,30 +285,98 @@ logging.info('Sorting events into template duration bins')
 n_bins = duration_bin_edges.size - 1
 alphas = {i: np.zeros(n_bins, dtype=np.float32) for i in args.ifos}
 counts = {i: np.zeros(n_bins, dtype=np.float32) for i in args.ifos}
-binned_sngl_stats = {ifo: {} for ifo in args.ifos}
+event_bins = {}
+times_to_prune = {ifo: [] for ifo in args.ifos}
+
+# Note: turn off logging for cluster_over_time which is quite verbose
+logger = logging.getLogger()
+# Remember current logging level
+level = logger.level
+logger.setLevel(logging.WARN)
```

Comment:
Do not like this here. If `cluster_over_time()` is detrimentally verbose, it means it should use `logging.debug()` rather than `logging.info()`.

Reply:
I've removed it

I think this is a similar situation to where I removed some logging out of the cuts module before.

The implementation is different in different places - here it is used repeatedly over smaller chunks, but in other places it is used (maybe) once, so the logging is appropriate there. Finding/fixing every usage is going to be difficult, so will separate that into a different PR